### DISCUSSION TITLE
到扫码完就卡主了 看conf里的cookie还是空的

### DISCUSSION THREAD

**Discussion by yelanghzong** [Johnserf-Seed/TikTokDownload#566] [Q&A]
![d71c6b4a5e5717002e3f70b2ee0d345](https://github.com/Johnserf-Seed/TikTokDownload/assets/86045480/d0e03939-a53c-4603-ba97-d16641ba54ad)
![77af36631253e8bd6600d4a131c6684](https://github.com/Johnserf-Seed/TikTokDownload/assets/86045480/94180435-b02a-429e-862b-3f3be368223f)
扫码了没反应 就卡着了

**Comment by chengazhen** [comment]
风控了, 换个账号

**Reply by sldtt** [reply]
更换账号后也是同样的问题，这是为什么呢？

**Reply by sean0157** [reply]
> 更换账号后也是同样的问题，这是为什么呢？

俺也一样

**Reply by xxxzsgxxx** [reply]
俺也一样
